.gitignore

```diff
@@ -1,14 +1,21 @@
-auth.json
-
-logs/combined.log
-logs/error.log
-
-playground/filter_words.js
-playground/test.json
-
-# directories
-.tmp.drivedownload
-.vscode
-.logs
-.content
-node_modules/
+auth.json
+
+logs/combined.log
+logs/error.log
+
+playground/filter_words.js
+playground/test.json
+
+# directories
+.tmp.drivedownload
+.vscode
+.logs
+.content
+node_modules/
+.vs/VSWorkspaceState.json
+.vs/slnx.sqlite
+.vs/slnx.sqlite
+src/auth.json

```

Comment:
whats going on here - there are multiple auth.json lines
dont ignore the gitignore file